cpp/src/slice2py/PythonUtil.cpp

```diff
@@ -2276,7 +2278,7 @@ Slice::Python::CodeVisitor::writeSeeAlso(const StringList& seeAlso, bool needsNe
 }
 
 void
-Slice::Python::CodeVisitor::writeDocstring(const ContainedPtr& p, Output& out, const optional<string>& generatedType)
+Slice::Python::CodeVisitor::writeDocstring(const ContainedPtr& p, Output& out, string_view generatedType)
```

Comment:
For this overload of `writeDocString`, `generated_type` was _always_ passed.
So making it a defaulted `optional` parameter is useless.